src/Config.php

```diff
@@ -308,18 +309,22 @@ protected function parseGeneral()
                 // as a regex-like string, and we switched to an array. If we find the old style, fall back to the defaults.
                 unset($general['accept_file_types']);
             }
-            // accept uppercase and lowercase file extensions
-            $general['accept_file_types'] = array_unique(
-                array_merge(
-                    $general['accept_file_types'],
-                    array_map(function ($extension) {
-                        return strtolower($extension);
-                    }, $general['accept_file_types']),
-                    array_map(function ($extension) {
-                        return strtoupper($extension);
-                    }, $general['accept_file_types'])
-                )
-            );
+
+            // To remove unacceptable / unwanted extensions from the list of Acceptable File Types
+            $removeFromAllowedFileTypes = explode('|', 'sh|asp|cgi|php|php3|ph3|php4|ph4|php5|ph5|phtm|phtml');
+
+            // Create a bag with lowercased extensions
+            $bag = Bag::from($general['accept_file_types']);
+            $bag = $bag->map(function ($key, $ext) use ($removeFromAllowedFileTypes) {
+                if (!in_array(mb_strtolower($ext), $removeFromAllowedFileTypes)) {
+                    return mb_strtolower($ext);
+                }
+            })->clean();
+
+            // Set accept_file_types, with the previous bag _and_ uppercased alternatives.
+            $general['accept_file_types'] = $bag->merge($bag->map(function ($key, $ext) {
+                return mb_strtoupper($ext);
+            }))->toArray();
```

Comment:
However `.pHp` will still pass this firewall. `strtolower` should be done while checking extensions, not while building the list (maybe change it in separate PR?)

Reply:
it actually does that too. See line 319..

Example: 

![Schermafbeelding 2019-04-05 om 13 00 51](https://user-images.githubusercontent.com/1833361/55623324-f3e99200-57a2-11e9-83db-781a09f48cf3.png)

Result: 

![Schermafbeelding 2019-04-05 om 13 01 32](https://user-images.githubusercontent.com/1833361/55623330-f9df7300-57a2-11e9-9269-c00cdca2465c.png)

Reply:
So, if jpg is on this list, will `.jPG` be accepted too?